src/res/translation/translation_de_DE.ts

```diff
@@ -1314,17 +1314,17 @@
     <message>
         <location filename="../../clientsettingsdlg.cpp" line="178"/>
         <source>This opens the driver settings of your sound card. Some drivers allow you to change buffer settings, others like ASIO4ALL let you choose input or outputs of your device(s). More information can be found on jamulus.io.</source>
-        <translation>Dies öffnet die Treiber-Einstellungen der Soundkarte. Einige Treiber erlauben die Änderung der Puffer-Einstellungen, andere wie z.B. ASIO4ALL ermöglichen es Dir, die Audio Ein- und Ausgänge Deiner angeschlossenen Audiogeräte auszuwählen. Weitere Informationen hierzu findest Du auf jamulus.io</translation>
+        <translation>Dies öffnet die Treiber-Einstellungen der Soundkarte. Einige Treiber erlauben die Änderung der Puffer-Einstellungen, andere wie z.B. ASIO4ALL ermöglichen es Dir, die Audio Ein- und Ausgänge Deiner angeschlossenen Audiogeräte auszuwählen. Weitere Informationen hierzu findest Du auf jamulus.io.</translation>
```

Comment:
Maybe better: "Öffnet die Treiber-Einstellungen der Soundkarte". 

But thanks for fixing and finding these issues.

Reply:
Yep, changed, thanks. Don't want to interfere too much with the existing translation. Just happened to think that the spacing thing would be noticable in the UI. :)

Reply:
Yep! Thanks.